docs/example-library/meltano-run/index.md

```diff
@@ -0,0 +1,120 @@
+# Get setup
+
+This examples shows to how to get up and running using `meltano run` with `tap-gitlab`, `target-postgres` and `dbt`.
+To get started with this example download the [meltano yml](/docs/example-library/meltano-run/meltano.yml) to a fresh directory and run:
+
+```shell
+meltano install
+```
+
+## Configure an extractor
+
+The installed meltano.yml already includes a configured instance of the `tap-gitlab` extractor for this example. You can view
+the configuration of the extractor by running:
+
+```shell !
```

Comment:
example of how to skip a shell block. Adding a ` !` preserves syntax highlighting but signals to mdsh that this block shouldn't included.